From: Marcin Owsiany <marcin@owsiany.pl>
Subject: [PATCH] debian/html-support-gone

Change the manpage to reflect the fact that HTML support is not present in the version shipped by Debian.
This is due to licensing issues.

Signed-off-by: Marcin Owsiany <marcin@owsiany.pl>

---
 xmotd.8 | 33 ++++-----------------------------
 1 file changed, 4 insertions(+), 29 deletions(-)

diff --git a/xmotd.8 b/xmotd.8
index 6a54563..3db88ae 100644
--- a/xmotd.8
+++ b/xmotd.8
@@ -64,9 +64,11 @@ from user's
 file in cases where they can also login via dialup.
 
 .LP
+This version of
 .B xmotd
-can display messages marked-up with HTML and xpm colour pixmap
-logos. Support for these must be configured at compile-time.
+cannot display messages marked-up with HTML and xpm colour pixmap
+logos, because the code that provided that functionality does not
+comply with Debian Free Software Guidelines.
 
 .SH OPERATION
 .LP
@@ -151,13 +153,6 @@ is compiled-in, xpm colour pixmaps may be substituted instead. See
 NOTES for additional details.
 
 .TP 8
-.BI \-browser " web-browser"
-
-specifies the path and filename of a web-browser to be used when an
-URL is clicked (HTML version only). By default, \fIweb-browser\fP is
-\fI"/usr/local/bin/netscape"\fP. See NOTES for additional details.
-
-.TP 8
 .BI \-help
 
 displays command-line options usage.
@@ -458,26 +453,6 @@ back to root) setting the correct permissions and ownership on
 indication that the user has logged out because \fBTakeConsole\fP has
 changed ownership of the console.
 
-The \fB-browser\fP option was originally called \fB-netscape\fP; it
-was renamed to be more generic. When initially run, the browser is
-invoked as:
-
-.nf
-    /usr/local/bin/netscape %s
-
-.fi
-
-where \fB%s\fP is replaced by the selected URL. Subsequent URLs will
-be displayed in the already running browser using the syntax:
-
-.nf
-    /usr/local/bin/netscape -remote openURL(%s)
-
-.fi
-
-You may substitute a browser of your choice for \fBnetscape\fP, if it
-supports this syntax.
-
 .SH BUGS
 
 There are no provisions for displaying embedded images in the HTML
-- 
tg: (d3dafc5..) debian/html-support-gone (depends on: upstream)
